src/pages/EditRequestPage.js

```diff
@@ -70,11 +81,19 @@ const defaultProps = {
     policyTags: {},
     parentReportActions: {},
     transaction: {},
+    isLoadingReportData: true,
 };
 
-function EditRequestPage({report, route, parentReport, policyCategories, policyTags, parentReportActions, transaction}) {
+function EditRequestPage({report, route, parentReport, policyCategories, policyTags, parentReportActions, transaction, isLoadingReportData, isSmallScreenWidth}) {
     const parentReportActionID = lodashGet(report, 'parentReportActionID', '0');
     const parentReportAction = lodashGet(parentReportActions, parentReportActionID, {});
+    const parentReportID = lodashGet(report, 'parentReportID', '0');
+
+    const isTransactionLoadingRoute = lodashGet(transaction, 'comment.isLoading', false);
+    const isTransactionLoading = lodashGet(transaction, 'isLoading', false);
+
+    const isDataLoading = isLoadingReportData || isTransactionLoading || isTransactionLoadingRoute || _.isEmpty(transaction);
```

Comment:
For checking if reportData is loaded, is it possible to use `WithReportOrNotFound`? So that we re-use existing functions?